backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/premultiplied_alpha.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from enum import Enum
+
+import numpy as np
+
+from nodes.impl.image_utils import as_target_channels
+from nodes.properties.inputs import EnumInput, ImageInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+class AlphaAssociation(Enum):
+    UNPREMULTIPLY_RGB = "Unpremultiply RGB"
+    PREMULTIPLY_RGB = "Premulitply RGB"
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:premultiplied_alpha",
+    description="Divide (or multiply) the RGB channels with the Alpha channel values of an image to go from a Straight Alpha to a Premultiplied Alpha image state.",
+    name="Premultiplied Alpha",
+    icon="CgMathDivide",
+    inputs=[
+        ImageInput(),
+        EnumInput(AlphaAssociation, label="Alpha Association"),
```

Comment:
This is confusing. It's not clear whether this input is supposed to set the alpha association of the input image or determine the alpha association of the output image.

I think it would be better to call the option "Straight -> Premultiplied" and "Premultiplied -> Straight" to make the operation clear.

Reply:
Done, update inbound!